db/migrate/20180603194914_rename_armenian_locale_key.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class RenameArmenianLocaleKey < ActiveRecord::Migration[5.1]
+  def up
+    User.where(language: "hy").update_all(language: "hye")
```

Comment:
Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations.